app/models/searchgov_domain.rb

```diff
@@ -11,6 +11,10 @@ def delay
     end
   end
 
+  def index_urls
+    SearchgovDomainIndexerJob.perform_later(self, delay)
```

Comment:
We're passing the value of the `delay` to the job, to avoid having to call that method and hit `robots.txt` each time the job is run.